pkg/controllers/management/node/controller.go

```diff
@@ -84,6 +84,18 @@ func (m *Lifecycle) setupCustom(obj *v3.Node) {
 	}
 }
 
+func (m *Lifecycle) provisionCustom(obj *v3.Node) (*v3.Node, error) {
+	m.setupCustom(obj)
+	newObj, err := v3.NodeConditionProvisioned.Once(obj, func() (runtime.Object, error) {
```

Comment:
i guess we have to separate the condition in here. For create we can use NodeConditionInitialized, and for update we have to use a different condition to represent that the node is updated.